editor.planx.uk/src/pages/FlowEditor/components/Team/types.ts

```diff
@@ -1,8 +1,9 @@
 import { Role, User } from "@opensystemslab/planx-core/types";
 import React, { SetStateAction } from "react";
 
-export type TeamMember = Omit<User, "teams" | "isPlatformAdmin"> & {
+export type TeamMember = Omit<User, "teams" | "isPlatformAdmin" | "email"> & {
```

Comment:
As with the previous comment on `AddNewEditorFormValues` - we're unintentionally looseing our types here (both sematically, and in terms of type-safety).

Is might be the case that an `ArchivedTeamMember` is what you're intending to create / model?

```ts
type TeamMember = ActiveTeamMember | ArchivedTeamMember
```